galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -78,6 +79,21 @@ class CollectionRemoteSerializer(LastSyncTaskMixin, pulp_viewsets.CollectionRemo
     last_sync_task = serializers.SerializerMethodField()
     created_at = serializers.DateTimeField(source='pulp_created', required=False)
     updated_at = serializers.DateTimeField(source='pulp_last_updated', required=False)
+    proxy_address = serializers.CharField(
```

Comment:
if possible, this should be `proxy_url` so that this endpoint remains forwards compatible with pulp when pulp breaks the url up into fields.